.webpack/webpack.base.js

```diff
@@ -201,8 +201,6 @@ module.exports = (env, argv, { SRC_DIR, ENTRY }) => {
         '@hooks': path.resolve(__dirname, '../platform/app/src/hooks'),
         '@routes': path.resolve(__dirname, '../platform/app/src/routes'),
         '@state': path.resolve(__dirname, '../platform/app/src/state'),
-        'dicom-microscopy-viewer':
```

Comment:
can you also add reasoning why we are removing this

Reply:
@wayfarer3130

Reply:
@wayfarer3130 can you let Ali know why we removed this? I know you said we didn't need it anymore.

Reply:
We weren't referencing it - the loading all happens through the browser dynamic import so that the main project does not depend on the internal libraries and build those into the main library.